<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>树莓派人脸追踪与识别——树莓派端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="show_video">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>821</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_status">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_name">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>370</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_id">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>430</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>识别状态：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_setting">
    <property name="geometry">
     <rect>
      <x>1095</x>
      <y>590</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>首选项</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string>首选项</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
